pytorch_lightning/trainer/logging.py

```diff
@@ -92,7 +92,7 @@ def process_dict_result(self, output, train=False):
         # ---------------
         # all keys not progress_bar or log are candidates for callbacks
         callback_metrics = {}
-        if output:
+        if isinstance(output, dict):
```

Comment:
```suggestion
        if isinstance(output, Mapping):
```

also the import for Mapping is not available so far, but I cannot add it as a suggestion :)